pkg/machines/components/deleteDialog.css

```diff
@@ -11,3 +11,50 @@ table.delete-dialog-disks.table tbody tr td:nth-child(1) {
 table.delete-dialog-disks.table tbody tr td:nth-child(3) {
     text-align: right;
 }
+
+.bootstrap-select:not([class*=col-]):not([class*=form-control]):not(.input-group-btn) {
```

Comment:
Aren't these changes related to the CreateVmDialog? I think it's better to keep the code isolated from the DeleteDialog.
For generic changes, please use `machines.less`.